crates/node/src/parse.rs

```diff
@@ -182,7 +180,7 @@ pub fn parse_file_sync(cx: CallContext) -> napi::Result<JsString> {
 pub fn parse_file(cx: CallContext) -> napi::Result<JsObject> {
     let c = get_compiler(&cx);
     let path = PathBuf::from(cx.get::<JsString>(0)?.into_utf8()?.as_str()?);
-    let options = cx.get::<JsString>(1)?.into_utf8()?.as_str()?.to_string();
+    let options: ParseOptions = cx.get_deserialized(1)?;
```

Comment:
This is executed in js thread.
That's why I used `String`

Reply:
does execution context related to what type of value we'll try to grab? since parse_file passes object, I thought it requires serialization other than coerce into string and current error is due to current path attempts to deserialize object into string.

Reply:
If so, we should pass it as a JSON form and store it to the task as bytes/string and parse from the background thread.